core/api/models.py

```
@@ -1,3 +1,81 @@
 from django.db import models
+from uuid import uuid4
+from auth_app.models import CustomUser
+
+
+
+class contribution_videos(models.Model):
+    """
+    Model for storing contribution videos.
+    """
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    title = models.CharField(max_length=255,null=True,blank=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `CharField`

(DJ001)